knowledge-base/telerik-reporting-r3-2016---upgrade-issues-with-objectdatasource-component.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Telerik Reporting R3 2016 - Upgrade Issues with ObjectDataSource Component
```

Comment:
Reverse this - Upgrade issues with ObjectDataSource Component after Telerik Reporting R3 2016